core/src/main/java/org/gbif/pipelines/core/io/DwCAReader.java

```diff
@@ -21,13 +21,19 @@
  */
 public class DwCAReader {
```

Comment:
add javadoc to both constructors to make it clear when one should be used instead the other